projects/adapter/pyproject.toml

```diff
@@ -34,6 +34,7 @@ google-cloud-bigquery = { version = "~3.5.0", extras = ["pandas"], optional = tr
 
 ## redshift
 sqlalchemy-redshift = { version = "^0.8.9", optional = true }
+awswrangler = { version = ">=3.0.0", extras = ["redshift"], optional = true}
```

Comment:
also curious if you considered the one used by dbt-redshift?

https://github.com/dbt-labs/dbt-redshift/blob/b644963a6bf7d3e64bd71e6ee97a205900089db1/setup.py#L87-L88

Reply:
Yeah, thank you for suggesting, @chamini2! awswrangler just seemed tidier. They also have pretty good Athena support, so I was thinking that maybe we use it for Athena as well.

Reply:
https://github.com/aws/aws-sdk-pandas

Reply:
Very interesting! Let's try that!